symplyphysics/laws/electricity/ampere_law.py

```diff
@@ -0,0 +1,51 @@
+from sympy import (Eq, solve, cos)
+from symplyphysics import (
+    units,
+    Quantity,
+    Symbol,
+    print_expression,
+    validate_input,
+    validate_output,
+    dimensionless
+)
+
+# Description
+## Ampere's law is the law that determines the force with which a magnetic field acts on
+## a small segment of a conductor with a current. The force turns out to be linearly dependent 
+## on both current and magnetic induction.
+
+## Law is: F = I * L * B * cos(a), where
+## F - force,
+## I - current,
+## L - length of conductor,
+## B - magnetic induction,
+## a - angle between magnetic induction and current direction.
+
+force = Symbol("force", units.force)
+
+current = Symbol("current", units.current)
+length = Symbol("length", units.length)
+induction = Symbol("induction", units.magnetic_density)
+angle = Symbol("angle", dimensionless)
+
+law = Eq(force, current * length * induction * cos(angle))
+
+
+def print_law() -> str:
+    return print_expression(law)
+
+
+@validate_input(current_=current,
+    length_=length,
+    angle_=angle,
+    induction_=induction)
+@validate_output(force)
+def calculate_force(current_: Quantity, length_: Quantity, angle_: float, induction_: Quantity) -> Quantity:
```

Comment:
You can accept angle as both Quantity and float, because Quantity allows some angle conversions.